### DISCUSSION TITLE
Python Italic/Bold Highlight Groups

### DISCUSSION THREAD

**Discussion by b4rlw** [NvChad/NvChad#2128] [Q&A] [answered]
I would like some italic/bold highlight groups as shown in the attached image, but for Python code instead.

![149852082-279e8a8a-e1e0-4b94-90a5-afeb519cd1b3](https://github.com/NvChad/NvChad/assets/47506872/a55698f5-98b2-4e8e-8bb5-87f9c5a3c42c)

I'm struggling to get anything changed except functions and comments, but would like the ability to control operators, keywords, etc.

Has anyone done this successfully?

Cheers.

**Comment by siduck** [comment, marked as answer]
@b4rlw please check the list of all highlight groups used by  nvchad in base46 repo,check the treesitter/syntax file especially

**Reply by b4rlw** [reply]
Got things the way I wanted them for Python with the following `highlights.lua`:

```lua
local M = {}

---@type Base46HLGroupsList
M.override = {
  -- imports
  Include = {
    italic = true,
  },
  -- for, while loops
  Repeat = {
    italic = true,
  },
  -- if statements
  Conditional = {
    italic = true,
  },
  -- class, with, as keywords
  ["@keyword"] = {
    italic = true,
  },
  -- function keyword
  ["@keyword.function"] = {
    italic = true,
  },
  -- return, yield keywords
  ["@keyword.return"] = {
    bold = true,
  },
  -- function names
  ["@function"] = {
    italic = false,
  },
  -- symbolic
  ["@operator"] = {
    bold = true,
  },
  -- logical keywords 
  ["@keyword.operator"] = {
    italic = true,
  },
  -- function parameters
  ["@parameter"] = {
    italic = true,
  },
}

return M
```

**Reply by siduck** [reply]
@b4rlw i dont think there's much u could do! but i have a workaround. i'll help you tomorrow
